integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/action/recipe/RecipeTestClient.java

```diff
@@ -0,0 +1,53 @@
+package com.sequenceiq.it.cloudbreak.newway.action.recipe;
+
+import static com.sequenceiq.it.cloudbreak.newway.log.Log.log;
+import static com.sequenceiq.it.cloudbreak.newway.log.Log.logJSON;
+import static java.lang.String.format;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.sequenceiq.it.cloudbreak.newway.CloudbreakClient;
+import com.sequenceiq.it.cloudbreak.newway.context.TestContext;
+import com.sequenceiq.it.cloudbreak.newway.entity.recipe.RecipeTestDto;
+
+public class RecipeTestClient {
```

Comment:
Please take a look at `com.sequenceiq.it.cloudbreak.newway.client.LdapConfigTestClient`.
These post and delete calls should be implemented inside actions and client should return actions.